contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_equal_to_single_column.py

```diff
@@ -73,10 +81,13 @@ class ExpectMulticolumnSumValuesToBeEqualToSingleColumn(MulticolumnMapExpectatio
 
     Args:
         column_list (list of str): \
-            A list of 2 or more integer columns, in which we expect the sum of the first n-1th \
+            A list of 2 or more float columns, in which we expect the sum of the first n-1th \
```

Comment:
It appears that integers would also work, how about changing to:
```suggestion
            A list of 2 or more int or float columns, in which we expect the sum of the first n-1th \
```